alf/environments/carla_sensors.py

```diff
@@ -640,6 +640,10 @@ def _to_numpy_loc(loc: carla.Location):
     return np.array([loc.x, loc.y, loc.z], dtype=np.float)
 
 
+def _to_carla_loc(loc):
```

Comment:
This function seems not used anywhere.

Reply:
I used it when I wrote debugging code. It doesn't hurt to keep it.